packages/components/psammead-most-read/README.md

```diff
@@ -180,7 +186,7 @@ import Timestamp from '@bbc/psammead-timestamp';
 
 ### When to use these components
 
-These components are intended to be used on `article` and `front` pages.
+These components are intended to be used on `article`, `front` pages.
```

Comment:
We also have most read on its own page now: https://test.bbc.co.uk/pidgin/popular/read
I've specified it as `mostRead` pagetype in simorgh: https://github.com/bbc/simorgh/blob/latest/src/app/routes/mostRead/index.js#L10

```suggestion
These components are intended to be used on `article`, `front` and `mostRead` pages.
```

Reply:
and do we wanna mention STY's here as well 🤔